Modules/Microsoft365DSC/DSCResources/MSFT_MEMEndpointProtectionApplicationControlPolicyWindows10/MSFT_MEMEndpointProtectionApplicationControlPolicyWindows10.psm1

```diff
@@ -0,0 +1,495 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter(Mandatory = $True)]
+        [System.String]
+        $DisplayName,
+
+        [Parameter()]
+        [System.String]
+        $Description,
+
+        [Parameter()]
+        [ValidateSet("notConfigured", "enforceComponentsAndStoreApps","auditComponentsAndStoreApps", "enforceComponentsStoreAppsAndSmartlocker", "auditComponentsStoreAppsAndSmartlocker")]
+        [System.String]
+        $AppLockerApplicationControl,
+
+        [Parameter()]
+        [System.Boolean]
+        $SmartScreenBlockOverrideForFiles,
+
+        [Parameter()]
+        [System.Boolean]
+        $SmartScreenEnableInShell,
+
+        [Parameter(Mandatory = $True)]
+        [System.String]
+        [ValidateSet('Absent', 'Present')]
+        $Ensure,
+
+        [Parameter()]
+        [System.Management.Automation.PSCredential]
+        $Credential,
+
+        [Parameter()]
+        [System.String]
+        $ApplicationId,
+
+        [Parameter()]
+        [System.String]
+        $TenantId,
+
+        [Parameter()]
+        [System.String]
+        $ApplicationSecret,
+
+        [Parameter()]
+        [System.String]
+        $CertificateThumbprint
+    )
+
+    Write-Verbose -Message "Checking for the MEM Endpoint Protection Application Control Policy {$DisplayName}"
```

Comment:
Also update verbose message to refer to Intune. This will be reviewed across all resources if a rebranding becomes necessary in the future.

Reply:
Updated